jobs/pacman-data-shipper/src/main/java/com/tmobile/cso/pacman/datashipper/config/ConfigManager.java

```diff
@@ -1,81 +1,76 @@
 package com.tmobile.cso.pacman.datashipper.config;
 
-import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
-
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
 import com.fasterxml.jackson.core.type.TypeReference;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.tmobile.cso.pacman.datashipper.dao.RDSDBManager;
 import com.tmobile.cso.pacman.datashipper.util.Constants;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
+import java.io.IOException;
+import java.util.*;
 
 /**
```

Comment:
Let's remove such comments - they are purely useless.
Same in other files. They just take space and brings no value.